developer-docs-site/docs/reference/glossary.md

```diff
@@ -187,6 +187,19 @@ then there is a guarantee that T_N will never be included in the blockchain.
 
 - **Fullnodes** are clients that ensure data are stored up-to-date on the network. They replicate blockchain state and transactions from other fullnodes and validator nodes.
 
+### Fungible Asset
+
+- A **fungible asset** is an asset, such as a currency, share, in-game resource, etc., that is interchangeable with another identical asset without any loss in its value. For example, APT is a fungible asset because you can exchange one APT for another.
+- Follow the [Digital Asset Standards](../standards/index.md#digital-asset-standards) to create fungible assets on the Aptos blockchain. 
+
+### Fungible Token
+
+- A **fungible token** is a token that is interchangeable with other identical tokens (i.e., tokens that share the same `TokenId`). This means the tokens have the same `creator address`, `collection name`, `token name`, and `property version`.
```

Comment:
this is different in token v2. fungible token should be a special type of fungible asset that the asset metadata is a token.

Reply:
I defined v1 first since the v2 docs are not public yet. I'm okay with updating the definition in this PR to include v1 and v2, then waiting until the other docs land before landing this PR.

Reply:
Do we want to keep it in general?

Reply:
agreed, let's define two bullets here.